### PR TITLE
Filter Sensor

### PR DESCRIPTION
**Pull request by dgomes**
**Description:**

Documentation for the filter component 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12650

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

### CODE REVIEW

File: source/_components/sensor.filter.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.12
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+The `filter` component supports sensors which process the **numeric states** from other entities. `filter` will apply a signal processing algorithm to a serie of states and produce a `new state` given the choosen algorithm.
```
Comment (c727) [inline source/_components/sensor.filter.markdown]: series
produces/ generates/ calculates

File: source/_components/sensor.filter.markdown [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    name: filtered relative humidity 
+    filter: lowpass
+    options:
+      time_constant: 10
+  - platform: filter
+    entity_id: sensor.temperature
+    name: filtered temperature
+    filter: outlier
+    options:
+      constant: 0.10
```
Comment (c727) [inline source/_components/sensor.filter.markdown]: is this time_constant? it's not listed again in the details below

File: source/_components/sensor.filter.markdown (hunk 1 above)
Comment (dgomes) [reply source/_components/sensor.filter.markdown]: How about:

```
`filter` will apply a signal processing algorithm to a series of states and generate a `new state` given the chosen algorithm.
```

I changed to *series* (plural), but each filter works with a single serie (singular). I'm no english speaking native, but it made sense to me. Tell me how to proceed.

File: source/_components/sensor.filter.markdown (hunk 2 above)
Comment (dgomes) [reply source/_components/sensor.filter.markdown]: no... it's **radius**... will correct

**Pull request merged**